### PR TITLE
Update description of device report view

### PR DESCRIPTION
**Pull request by ravisha22**
Added description to provide more information on the data being displayed at the device report

### CODE REVIEW

**Comment by PRMerger16** [comment]
@ravisha22 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: windows-365/enterprise/report-remoting-connection.md [hunk 1]
```diff
@@ -81,7 +81,10 @@ You can review the **Round trip time** and **Sign in time** for each Cloud PC de
 
 ## Device history
 
-When you select on a particular device in one of the reports, you'll see specific information for that device.  
+When you select on a particular device in one of the reports, you'll see specific information for that device. The information displayed is for the last 14 days of usage with values aggregated for each day individually. If the user has not logged in to the Cloud PC for a particular day, those days will not show in the graph.
+>[Note]
+>There might days when users login and the graph does display those days. This is due to the way the graph calculations exclude certain edge cases that can provide wrong data.
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/report-remoting-connection.md:86]:
```suggestion
When you select a particular device in one of the reports, you'll see specific information for that device. The information displayed is for the last 14 days of usage with values aggregated for each day individually. If the user has not signed in to Cloud PC for a particular day, that day will not show in the graph.

> [!NOTE]
> There might be days when users have signed in and the graph does not display those days. This is due to the way the graph calculations exclude certain Edge cases that can provide incorrect data.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ravisha22, kindly commit the suggested changes. Many thanks.

**Comment by PRMerger-2** [comment]
@ravisha22 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ErikjeMS** [comment]
#sign-off

**Comment by ErikjeMS** [comment]
This section was already updated in a PR on the private repo. So, closing this PR.

**Pull request closed without merging**